src/handlers/value-inspectors/primitive-value-inspector.ts

```diff
@@ -29,14 +29,10 @@ export class PrimitiveValueInspector implements ValueInspector {
   }
 
   public deduceValue<T>(propertyDto: PropertyDto): any {
-    if (propertyDto.value) {
+    if (propertyDto.value !== 'undefined') {
```

Comment:
remove

Reply:
We need it, it's very long to explain the case but I will show you on video